ranger/ext/human_readable.py

```diff
@@ -19,42 +18,61 @@ def human_readable(byte, separator=' '):  # pylint: disable=too-many-return-stat
     '1023 M'
     """
 
+    units = ['B', 'K', 'M', 'G', 'T', 'P']
+
     # handle automatically_count_files false
     if byte is None:
         return ''
 
-    if SettingsAware.settings.size_in_bytes:
-        return format(byte, 'n')  # 'n' = locale-aware separator.
+    # if SettingsAware.settings.size_in_bytes:
+    #     return format(byte, 'n')  # 'n' = locale-aware separator.
+
+    [formatted_number, unit_index] = _preformat_bytes(byte)
+    if unit_index is None:
+        return formatted_number
+    return '{0:s}{1:s}{2:s}'.format(formatted_number, separator, units[unit_index])
 
+def _preformat_bytes(byte):  # pylint: disable=too-many-return-statements
```

Comment:
Maybe we should rename `byte` everywhere? It's a bit confusing cause it's not a byte we're storing.

Reply:
That's a good point. Renamed in 322960b.